modules/nf-core/samshee/tests/main.nf.test.snap

```diff
@@ -78,30 +78,30 @@
                             "id": "test",
                             "lane": 1
                         },
-                        "SampleSheet_formatted.csv:md5,9a1cac9e958256a17c7f43a8e15cb697"
+                        "SampleSheet_formatted.csv:md5,9cec8920fa7178fe82af87e2cf50a3cb"
                     ]
                 ],
                 "1": [
-                    "versions.yml:md5,cbbf34760bb4db290b14d619e7844f0e"
+                    "versions.yml:md5,fc78ab25e327168ba8875015db743a31"
                 ],
                 "samplesheet": [
                     [
                         {
                             "id": "test",
                             "lane": 1
                         },
-                        "SampleSheet_formatted.csv:md5,9a1cac9e958256a17c7f43a8e15cb697"
```

Comment:
Any idea what changed that caused not just the version snapshot to change?

Reply:
I inspected the files because I noticed the same and I did not see any visible differences, so I am not sure.